src/components/SimpleInput.js

```diff
@@ -0,0 +1,55 @@
+const React = require('react');
+const Radium = require('radium');
+
+const StyleConstants = require('../constants/Style');
+
+const Input = React.createClass({
+  propTypes: {
+    name: React.PropTypes.string,
+    placeholder: React.PropTypes.string,
+    type: React.PropTypes.string,
+    valid: React.PropTypes.bool
+  },
+
+  getDefaultProps () {
+    return {
+      type: 'text',
+      valid: true
+    };
+  },
+
+  render () {
+    const styles = this.styles();
+
+    return (
+      <input
+        {...this.props}
+        name={this.props.name}
+        style={Object.assign({}, styles.wrapper, this.props.style)}
+        type={this.props.type}
```

Comment:
Yes I would drop name but keep style and type.  Type could then just be set to `text` rather than using a prop to set it.  This would ensure that the input stays as type text even if the user of the component passes in a type prop as our specified type overwrites the type coming in with the `...this.props` spread.  

Unless I'm miss understanding what we're trying to achieve with enforcing type as text.